### DISCUSSION TITLE
Task not showing up in planner

### DISCUSSION THREAD

**Discussion by imageless-singularity** [ivan-lednev/obsidian-day-planner#552] [Q&A]
While the first task I put in the planner (on the planner itself, like dragging down) shows up, subsequent entries are there but disappear once I click off them, even though they appear in the daily planner note. Am I doing something wrong?

**Comment by ivan-lednev** [comment]
Hey @imageless-singularity, please follow the issue template for this. There is not enough detail for me to analyze the issue.
https://github.com/ivan-lednev/obsidian-day-planner/issues/new?assignees=&labels=bug&projects=&template=bug_report.yaml
